cob/cli/docker_cli.py

```diff
@@ -208,6 +209,8 @@ def _exec_docker_compose(cmd, **kwargs):
     with open(compose_filename, 'w') as f:
         f.write(_generate_compose_file(**kwargs))
     docker_compose = shutil.which('docker-compose')
+    if not docker_compose:
+        raise MissingDependency("No 'docker-compose' executable was found")
```

Comment:
Perhaps "docker-compose is not installed in this system. Please install it to use cob"?